lib/http/uri.rb

```diff
@@ -36,14 +36,15 @@ def self.parse(uri)
       new(Addressable::URI.parse(uri))
     end
 
-    # Encodes key/value pairs as application/x-www-form-urlencoded
+    # Encodes key/value pairs as a URL query string
     #
-    # @param [#to_hash, #to_ary] form_values to encode
-    # @param [TrueClass, FalseClass] sort should key/value pairs be sorted first?
+    # @param [#to_hash, #to_ary] values to encode
     #
-    # @return [String] encoded value
-    def self.form_encode(form_values, sort = false)
-      Addressable::URI.form_encode(form_values, sort)
+    # @return [String] encoded query string
+    def self.encode_query(values)
```

Comment:
Please do not remove `form_encode` method in this PR. I'm fine if we will drop this but this is a backward compatibility breakage. And I think it was added here for some reason back in days.

@tarcieri do you remember if we actually need this method to be available on `HTTP::URI` or is it OK to completly remove it?

Reply:
I assumed it was only used internally. I never considered this gem providing a URI class for external usage, but that makes sense if folks use it that way.